demo/fiber/iotdb-sync-stress-demo/README.md

```diff
@@ -0,0 +1,14 @@
+dependency:
+1. iotdb
```

Comment:
Can you update iotdb's server setup process here?

Reply:
Okay, tonight I will supplement the creation process of the iotdb instance in as much detail as possible

Reply:
I have updated iotdb's server and cli setup process in the new commit